test/inspect.test.ts

```diff
@@ -757,6 +757,23 @@ if (goVersion[0] > 1 || goVersion[1] >= 12) {
       t.deepEquals(pkg, expectedDepTree);
     });
   });
+
+  test('invalid go.mod', {timeout: 120000}, async (t) => {
```

Comment:
If the test we have is for failing to parse `go.mod` file, can we do away with most of the `gomod-small-failing` fixture? Including the expected-tree and graph.

Reply:
yep - I'll take out everything except the go.mod